pkg/gardenlet/controller/shoot/shoot_control_reconcile.go

```diff
@@ -190,10 +190,25 @@ func (c *Controller) runReconcileShootFlow(o *operation.Operation, operationType
 			Fn:           flow.TaskFn(botanist.WaitUntilControlPlaneReady),
 			Dependencies: flow.NewTaskIDs(deployControlPlane),
 		})
+		syncETCDEncryptionConfiguration = g.Add(flow.Task{
+			Name:         "Syncing etcd encryption configuration to ShootState",
```

Comment:
I'll check how I can optimise these, generally I'd like to keep `Syncing etcd encryption configuration to ShootState` as a separate step as it will be really simple to remove in the near future (we won't need it once all shoots have updated the encryption metadata to their shootstates.